sonar-go-checks/src/test/resources/checks/DuplicatedFunctionImplementationCheck/duplicated_function_implementation.go

```diff
@@ -0,0 +1,89 @@
+package DuplicatedFunctionImplementationCheck
+
+func foobar() {}
+func foo_bar () int {}  // Compliant - has no line
+
+func smallF1() {
+  foo = 1;
+  bar = foo > 3 || bar
```

Comment:
Compilation error: `undefined: bar`.
I would expect the examples that are compilable, please adjust the functions in the file.

Reply:
I missed a lot of things about go, thanks for pointing out.
I was thinking that we were using sort of strict Go compiler to create the AST, I guess I was wrong :sweat_smile: